### DISCUSSION TITLE
WebdriverIO browser.url("urlpath") not working with Nodejs version  > 12.18.0

### DISCUSSION THREAD

**Discussion by sachin-agarwal-by** [webdriverio/webdriverio#7914] [Q&A]
Does webdriverio doesnot work with Nodejs version > 12.18.0 because when I have Nodejs version > 12.18.0 and using browser.url("urlpath") in the code then browser is not opening with the url path

**Comment by christian-bromann** [comment]
WebdriverIO supports Node.js v12 and upwards. If the `url` command doesn't work, check that it is called and you are using `async/await` properly. Use WebdriverIOs [debugging capabilities](https://webdriver.io/docs/debugging/) to find out what is going on.

**Reply by sachin-agarwal-by** [reply]
@christian-bromann I am not using async and await pattern across code but it work with NodeJs 12  version.. Can you suggest some example for async/await if its necessary to implement.

**Reply by christian-bromann** [reply]
You should use `async/await`. Please check out some of the [boilerplate projects](https://webdriver.io/docs/boilerplates) from the docs.

**Reply by sachin-agarwal-by** [reply]
these are webdriver v7 boiler project but I had used version 5 webdriverio and taken https://github.com/ssehmi/e2e boilerproject which was working with NodeJs < 12.18.0 but not with greater version, any suggestion or we have to upgrade the webdriverio version if we move to Nodejs higher version ? @christian-bromann

**Reply by christian-bromann** [reply]
It is not recommended to use a deprecated WebdriverIO version. Please use the latest.
